topi/python/topi/nn/batch_norm.py

```diff
@@ -54,3 +54,33 @@ def batch_norm_inference(data, gamma, beta, moving_mean, moving_var, eps, fix_ga
     mean = tvm.compute((C, ), lambda c: moving_mean[c])
     var = tvm.compute((C, ), lambda c: moving_var[c])
     return [out, mean, var]
+
+@tvm.target.generic_func
+def batch_norm(data, moving_mean, moving_var, eps):
```

Comment:
the scale_shift operator already does this

Reply:
Done